src/spec-node/featuresCLI/testCommandImpl.ts

```diff
@@ -66,10 +67,27 @@ export async function doFeaturesTestCommand(args: FeaturesTestCommandInput): Pro
 		}
 	}
 
+	// Clean up test containers
+	if (!args.preserveTestContainers) {
+		await cleanup(cliHost);
+	}
+
 	// Pretty-print test results and exit with 0 or 1 exit code.
 	return analyzeTestResults(testResults);
 }
 
+async function cleanup(cliHost: CLIHost) {
+	// Delete any containers that have the 'devcontainer.is_test_run=true' label set.
+	const filterForContainerIdArgs = ['ps', '-a', '--filter', 'label=devcontainer.is_test_run=true', '--format', '{{.ID}}'];
```

Comment:
Let's say, I run tests with `preserve-test-containers` flag, then for the second time I run tests without it.

Won't this command also clean up containers from first run? 
I think this is okay for now, but thought of pointing it out.

Reply:
Yep, it will clean everything up.  I don't think that will cause any problems, it might actually be nice that subsequent runs do the clean up for you unless you keep on preserving.  I imagine this flag being used as you're manually iterating on a single test.